### PR TITLE
JBIDE-19537 Document some of known problems in VPE FAQ related to using WebKit based internal browser

### PR DESCRIPTION
**Pull request by kmarmaliykov**

### CODE REVIEW

**Comment by alexeykazakov** [comment]
Looks good to me.

File: documentation/faq/visualeditor.adoc [hunk 1]
```diff
@@ -55,7 +55,10 @@ Visual Part doesn't start and *"XPCOM error -2147467259"* is shown::
       * Go to Eclipse -> Help -> About -> Installation Details... ->  Configuration
       * find org.eclipse.swt.browser.XULRunnerPath and open this path in terminal
       * try xulrunner.exe -version or ./xulrunner -version and it should show what is needed for XR to work
-	
+
+After choosing HTML5 Visual Editor mode on Linux Eclipse freezes during restart, what can I do with it?::
+	This may happen because SWT.WebKit conflicts with http://bluejeans.com/[BlueJeans] plugin for Firefox. Also the cause of this problem may be http://www.google.com/hangouts/[Google+ Hangouts] browser plugin. Uninstalling this plugins will fix Eclipse freezeing.
```
Comment (maxandersen) [inline documentation/faq/visualeditor.adoc]: do we have link to related bugzilla ? 

"this plugins" -> "these plugins"

**Comment by xcoulon** [comment]
I just pushed the PR with the typo fix. Please open a new PR if you want to add links to related bugzilla.

File: documentation/faq/visualeditor.adoc (hunk 1 above)
Comment (kmarmaliykov) [inline documentation/faq/visualeditor.adoc]: yes, we do have related issues in redhat bugzilla. And I was actually wrong, with google talk plugin eclipse crashes, not freezes. I will update FAQ now.

**Comment by kmarmaliykov** [comment]
@maxandersen @xcoulon @alexeykazakov look into #371

**Pull request closed without merging**